contracts/src/deploy/deploy_plugin.ts

```diff
@@ -1,21 +1,28 @@
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
-import { getProtocolManagerAddress } from "../utils/protocol";
+import { getProtocolManagerAddress, getProtocolRegistryAddress } from "../utils/protocol";
 
 const deploy: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     const { deployments, getNamedAccounts } = hre;
     const { deployer } = await getNamedAccounts();
     const { deploy } = deployments;
 
-    const manager = await getProtocolManagerAddress(hre)
-    console.log({manager})
+    const registry = await getProtocolRegistryAddress(hre, deployer);
```

Comment:
I don't see `getProtocolManagerAddress` with this signature, only the one that accepts the HRE environment only. Am I missing something?